src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -32,6 +31,7 @@ private[outwatch] object DomTypesBuilder {
       //encodes true as "" and false as null, whereas snabbdom needs true/false
       //of type boolean (not string) for toggling the presence of the attribute.
       case _: BooleanAsAttrPresenceCodec.type => identity
+      case _: BooleanAsIsCodec.type => identity
```

Comment:
We need to make a different handling for attriibutes then. Because snabbdom will interpret a boolean value as a `boolean attribute` like `BooleanAsAttrPresenseCode` which would remove the attribute in case of a false value.

Reply:
I think we need to do if for all boolean `Codec`, as neither of them should be `toString` in case of a property.

Reply:
I'm not sure I know what you mean. Attributes and props are passed to snabbdom in different structures.

Reply:
The handling of attributes and properties is different with regards to boolean values: https://github.com/snabbdom/snabbdom/releases/tag/v0.7.0. Any value `true|false` for attribute will be interpreted as a boolean attribute. That is `<input checked>` corresponds to `checked=true` and <input> to `checked=false`.

So afaik we should really only make boolean values for attributes if they are `BooleanAsAttrPresenceCodec`, which signals being boolean attribute (it shouldn't even be there in case of properties). Does that help?